build/buildRunTimeImages.sh

```diff
@@ -43,13 +45,21 @@ fi
 docker build \
     --pull \
     -f "$RUNTIME_BASE_IMAGE_DOCKERFILE_PATH" \
-    -t "$RUNTIME_BASE_IMAGE_NAME" \
+    -t "$RUNTIME_BASE_IMAGE_NAME-stretch" \
+    --build-arg COMMON_RUNIMAGE_BASE=stretch \
+    $REPO_DIR
+
+docker build \
```

Comment:
This should not be required if you are getting the debian flavor as argument in line 18?

Reply:
We are building both buster and stretch at the same machine now .. so it doesn't harm to build the common base together

---
In reply to: [430621412](https://github.com/microsoft/Oryx/pull/592#discussion_r430621412) [](ancestors = 430621412)